### PR TITLE
Changelog 1.13.1

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+CHANGES IN VERSION 1.13.1 (FROM 1.13.0)
+===================================
+
+Appium 1.13.0 is a patch release. This [addresses problems with long startup times for Safari WebViews on iOS 12.2](https://github.com/appium/appium/issues/12590)
+
+#### Android (Espresso)
+* Fix problems with activity startup not working when specifying `appActivity` and `appPackage` [#441](https://github.com/appium/appium-espresso-driver/pull/441)
+
+#### iOS (XCUITest)
+* Make `platformVersion` a required capability [#954](https://github.com/appium/appium-xcuitest-driver/pull/954)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: Actually, it is required for Simulator. Appium will try to retrieve it using imobileinfo tool on real devices if the cap is not set explicitly

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+CHANGES IN VERSION 1.13.1 (FROM 1.13.0)
+===================================
+
+Appium 1.13.0 is a patch release. This [addresses problems with long startup times for Safari WebViews on iOS 12.2](https://github.com/appium/appium/issues/12590)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: 1.13.1

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+#### Android (Espresso)
+* Fix problems with activity startup not working when specifying `appActivity` and `appPackage` [#441](https://github.com/appium/appium-espresso-driver/pull/441)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: ... when the package name value in appActivity is different from the one in appPackage

**Comment by mykola-mokhnach** [comment]
should we also mention node10 is minimum supported?

**Comment by dpgraham** [comment]
Done

**Pull request closed without merging**